ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -366,6 +367,31 @@ export default function BuildQuote({
             defaultToAll
           />
         </div>
+        {selectedToToken?.address &&
+          selectedToToken?.address !== ETH_SWAPS_TOKEN_OBJECT.address && (
+            <div className="build-quote__token-message">
+              <span>
```

Comment:
What is the purpose of this `span`?

Reply:
Nothing :sweat_smile:  It has been removed